kernel_gateway/gatewayapp.py

```diff
@@ -308,7 +308,7 @@ def client_ca_default(self):
     @default('ssl_version')
     def ssl_version_default(self):
         ssl_from_env = os.getenv(self.ssl_version_env)
-        return self.ssl_version_default_value if ssl_from_env is None else int(ssl_from_env)
+        return ssl_from_env if ssl_from_env is None else int(ssl_from_env)
```

Comment:
This change is correct right? Because we shouldn't be setting the `ssl_version` value to `self.ssl_version_default_value` at all without having any info on the other `ssl_options`. and always default it to `None` unless there is an env value set. I say this because when none of the certificate attributes are in the env, we set the `ssl_options` to `None` in [line](https://github.com/jupyter/kernel_gateway/blob/19dc96d8fc74568b339248c4c5aad08da4b4c2e5/kernel_gateway/gatewayapp.py#L513). and further more if there are any of the cert attributes mentioned but didn't specify the ssl_version, then we default it to 5 in the else block.

Reply:
This makes sense.